conformance/client/lite-unary-config.yaml

```diff
@@ -20,5 +20,3 @@ features:
     # so that we can run them all three ways: suspend,
     # callback, and blocking.
     - STREAM_TYPE_UNARY
-  supportsTlsClientCerts: true
-  supportsHalfDuplexBidiOverHttp1: true
```

Comment:
I've removed the line about client certs since this property defaults to false. An error is occurring when this code tries to instantiate a client cert from the data in the conformance request. I haven't dug into it, and decided to just omit it for now.

It turns out that we actually _can't_ do half-duplex stuff over HTTP 1.1, because we don't actually know ahead of time, for a given bidi RPC, if it's half-duplex or full-duplex (in the framework, based on generated protobuf metadata). But the request body _must_ be marked as "duplex" if it _might be_ full-duplex. So have to mark the body as duplex for all bidi RPCs, and okhttp then disallows them to be used with HTTP 1.1.

Reply:
Do we want to comment these properties out with a `TODO` to know to come back to them instead of removing them? Up to you but might help identify remaining work since it sounds like there are several things to track down.